assets/js/steps/components/navbar.js

```diff
@@ -60,6 +62,13 @@ function NavBar({ actionHandler }) {
                         {context.machineState.get.user.name}
                     </span>
                 )}
+                {context.boxConfig.get.debugEnabled && (
```

Comment:
<img width="188" alt="image" src="https://user-images.githubusercontent.com/15377965/93806999-51e59000-fc4a-11ea-8170-2bfb79b87ee0.png">

A bit over the top perhaps? In all seriousness, it should probably be visible that it is in debug mode.